localization/english/events/event_localization/aeluran_story_events_l_english.yml

```diff
@@ -1,17 +1,17 @@
 ﻿l_english:
-  aeluran_story.001.title:1 "Crimson Visitors"
-  aeluran_story.001.desc:0 "Theres a commotion in your holding. Sisters of the Aeluran Order, known as the red witches by the common folk, have showed up on your doorstep and are demanding your council. They say the know of your discovery on your last expedition."
-  aeluran_story.001.ok:0 "Ill hear them in my council chambers, steward see them in" 
+    aeluran_story.001.title:1 "Crimson Visitors"
+    aeluran_story.001.desc:0 "As you begin to settle down for the night, a servant bursts into your chambers in a rush. Outside, you hear a grand commotion as servants and guards alike rush to welcome your unexpected new guests. Sisters of the Aeluran Order, known as the red witches by the common folk, have somehow learnt of your latest expedition and are demanding your council."
+    aeluran_story.001.ok:0 "I'll hear them in my council chambers. Send them in."
 
-  aeluran_story.002.title:1 "Crimson Visitors"
-  aeluran_story.002.desc:0 "You enter your council chambers to find the Aeluran Sisters waiting. They are two women in red, both with an ageless beauty. One of the women steps forward.\n\n\"M'[ROOT.Char.GetLadyLord] my name is [aeluran_matron.GetFirstNameNoTooltip]. You have the privilege of standing before Mistress [high_matriarch.GetFirstNameNoTooltip], High Matriarch of the Aeluran Weavers.\" \n\n At this introduction, the other women steps forward. She moves with an effortless grace, her face ageless and wise.\n\n \"My [ROOT.Char.GetLadyLord], I know our arrival is unexpected but we come to discuss a matter most important. I felt the release of divine energy, and finding you an Elf in Human lands confirms it. I know you have found the Sigil of the Realm Lord\" "
-  aeluran_story.002.ok:0 "continue..." 
+    aeluran_story.002.title:1 "Crimson Visitors"
+    aeluran_story.002.desc:0 "You enter the council room to find two Aeluran Sisters awaiting your arrival. Their embroidered dresses and ageless beauty render you speechless for a moment, your shock only interrupted by the younger of the two clearing her throat. \n\n\"M'[ROOT.Char.GetLadyLord|U], my name is [aeluran_matron.GetFirstNameNoTooltip]. You have been granted the privilege of standing before Mistress [high_matriarch.GetFirstNameNoTooltip], High Matriach of the Aeluran Weavers.\" \n\nOn cue, the other sister steps forward to meet your eyes, moving with an effortless grace, her youthful face somehow projecting wisdom and sagacity. \n\n\"My [ROOT.Char.GetLadyLord|U], I know our arrival is unexpected, but we come to discuss a matter of great importance. I felt the release of divine energy a short time ago, and finding you, an elf in human lands, confirms my suspicions. I know you have found the Sigil of the Realm Lord.\""
```

Comment:
"shock" feels too strong. maybe instead "your momentary stupor only interrupted when the younger of the two clears her throat."

Im also not positive we should describe either as "younger", the previous sentence even says they appear "ageless" as they are literally immortal elves. maybe instead "the less senior of the two" or "the apparent junior of the two" or even "the speaker for the pair"